### DISCUSSION TITLE
Why isn't my pull request eligible for Hacktoberfest?

### DISCUSSION THREAD

**Discussion by mixed-farming** [community/community#69012] [Pull Requests]
### Select Topic Area

Question

### Body

I've submitted a pull request for an open-source project, and I was expecting it to count towards my contributions for Hacktoberfest. However, it seems that my pull request is not being recognized as part of the event. Can you please help me understand the specific reasons or criteria that might be preventing this pull request from being eligible for Hacktoberfest participation? I want to ensure that my contributions are in line with the requirements and guidelines of the event.

**Comment by SangeethaPrabhu2021** [comment]
**Go to the desired repository, check the About section**
![image](https://github.com/community/community/assets/86423721/5cfbab60-5daf-422d-862d-8cdb5ca1710e)
_If the tags arent present, that repo is not eligible for Hacktober fest. It should contains tags like Hacktoberfest Hacktoberfest2023 Hacktoberfest-accepted_

**Are you a maintainer, in that case you can add the tags in the following manner.**
![image](https://github.com/community/community/assets/86423721/938d5a82-08c3-4ec4-938a-de0e8a1a9be9)

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
